js/private/image/index.ts

```diff
@@ -28,15 +29,54 @@ type Entries = { [path: string]: Entry }
 
 type Compression = 'gzip' | 'none'
 
-function findKeyByValue(entries: Entries, value: string): string {
+function findKeyByValue(entries: Entries, value: string): string | undefined {
     for (const [key, { dest: val }] of Object.entries(entries)) {
         if (val == value) {
             return key
         }
     }
-    throw new Error(
-        `couldn't map ${value} to a path. please file a bug at https://github.com/aspect-build/rules_js/issues/new/choose`
-    )
+    return undefined
+}
+
+async function isSymbolicLink(filep: string, root: string) {
```

Comment:
This seems to return a string and not a boolean like I'd expect an `is*` method to. Can we rename this?

Reply:
agreed. renaming to get